app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixAvailabilityBottomSheet.kt

```diff
@@ -60,24 +64,28 @@ fun QuickFixAvailabilityBottomSheet(
                   CalendarView(
                       useCaseState =
                           rememberUseCaseState(
-                              visible = true,
-                              onCloseRequest = {
-                                if (clearEnabled) {
-                                  onClearClick()
-                                }
-                                onDismissRequest()
-                              }),
+                              visible = true, onCloseRequest = { onDismissRequest() }),
                       config =
                           CalendarConfig(
                               yearSelection = true,
                               monthSelection = true,
                               style = CalendarStyle.WEEK,
                           ),
                       selection =
-                          CalendarSelection.Dates { newDates ->
-                            onOkClick(newDates, timePickerState.hour, timePickerState.minute)
-                            onDismissRequest()
-                          })
+                          CalendarSelection.Dates(
+                              negativeButton =
+                                  SelectionButton(
+                                      textRes = R.string.clear, type = ButtonStyle.TEXT),
+                              onNegativeClick = {
+                                if (clearEnabled) {
+                                  onClearClick()
+                                  Log.d("chill guy", "dontshow")
```

Comment:
I hate to be that guy 😢, but change the `chill guy` comment otherwise it would be hard to understand if needed ;)

Reply:
But i'm a lowkey chill guy. I'll be right on it goat.